Sources/PackageGraph/DependencyResolver.swift

```diff
@@ -26,6 +26,9 @@ public enum VersionSetSpecifier: Equatable {
     /// A non-empty range of version.
     case range(Range<Version>)
 
+    /// The unit or singleton set.
+    case unit(Version)
```

Comment:
Ok, makes sense. Let's call this something else though, like `exact` or `individual` (some complement to `range`).